### DISCUSSION TITLE
Blackwing Lair PTR Tuning Feedback

### DISCUSSION THREAD

**Discussion by Annamaria-CC** [chromiecraft/chromiecraft#3204] [PTR Feedback]
You can provide your feedback for the PTR Tuning of BWL here. Please note that currently the tuning is in starting stages and will change often.

**Comment by Serakh90** [comment]
While I can't offer concrete feedback to the BWL tuning in place, I would like to offer my point of view on tuning content and the process behind it.

In general, tuning is in a weird spot in relation to CC goals, namely creating a blizzlike 3.3.5 experience. So I totally get that tuning is just a temporary thing just to make players happy in the moment, and that it contributes little to AC improvement besides getting people to actually play and test stuff.

I strongly disagree with some viewpoints that were brought up in the Discord channel, namely, using Classic logs as a basis for boss tuning. Using these numbers, maybe making the fights take 20% longer or the tanks take 10% more damage will just lead to players doing the fights exactly the way they did in Classic. This includes bypassing a lot of mechanics (off the top of my head: kiting adds on Razorgore, tank swaps on Vael, the whole stack mechanic on Firemaw, I can't even remember what differences Ebonroc and Flamegor are supposed to have). As a counter-example, I really did like the tuning of Beast in UBRS. With Classic numbers, it is a beyond boring fight, and a little more HP and melee damage would never have changed that. Turning the fire damage up to 11 and potentially globalling tanks made people experiment with tactics, positioning, gear etc, and I am sure this led to a way more thorough testing than it otherwise would have.

So, because it was brought up in the Discord discussion, I will use Broodlord as an example.  I don't think having Broodlord Mortal Strike potentially one-shot tanks is bad tuning necessarily. You could totally make it so MS can only be survived with defensive CDs, and make tanks rotate (there even is the Knock Away threat drop to facilitate this). If it is transparent and consistent when he uses MS and his other abilities, players can and will play around that, as they did with Beast. Have him MS twice within 10 seconds, then Knock Away, then 30 seconds pause, for example. 

As far as the actual testing is concerned, I am not sure I agree with doing all the tuning on a separate PTR. I cannot get the guild I play with to commit raid time to PTR testing (since well, a lot of people like to play their character and experience raids with it), and I don't want to join the more commited part of the playerbase on PTR, in part because of the extra effort to seek out and join such an in-group, and in part because I actually want to explore the tuning with the guild I raid with. In essence, I dislike the two-stage  testing model of PTR and Live, and I would much prefer if CC rolled out a somewhat buggy and badly tuned BWL on the live server for all people to test and report bugs, instead of doing the heavy lifting on a separate server. Because, from my perspective, there is (subjectively, of course) a HUGE hurdle for casual players to join PTR, but a new raid on Live is just a new raid on Live.

On a sidenote, did you ever consider releasing raids on a boss-by-boss basis? I would expect higher-quality feedback if there was a new BWL boss every, say, 2-3 weeks or when the previous one works fully as intended, so players and testers could focus on these few new mechanics specifically, instead of releasing all at once and getting a huge and diluted mess of feedback in return.

**Comment by balleny** [comment]
The note with a "gated" release of a dungeon surely sounds interesting, similar as it was done in retail with ICC, where bosses got available in batches. This would work well for testing / reporting I guess. But, i think the majority of player base might not be so happy with the gated release - especially the people who care (?!) about a realm first.
I would oppose a 1 boss per week release cycle, but maybe a 3-3-2 release group would be interesting.

On the other hand, there is a reason blizzard did not continue with this approach.
